docs/api.rst

```diff
@@ -828,6 +883,64 @@ Query the current exchange rate for select assets
    :statuscode 500: Internal Rotki error
    :statuscode 502: An external service used in the query such as cryptocompare/coingecko could not be reached or returned unexpected response.
 
+Query the historical price of assets
+======================================
+
+.. http:get:: /api/(version)/assets/prices/historical
+
+   Querying this endpoint with a list of asset\/timestamp lists and a target asset will return a dictionary with the price of the assets at the given timestamp in the target asset currency. Providing an empty list or no target asset is an error.
```

Comment:
`list of asset\/timestamp lists` -> list of lists of asset and timestamp.

dictionary -> object. Dictionary is python lingo. This is json and a dictionary is an object.